src/summary_statistics/mod.rs

```diff
@@ -45,16 +49,32 @@ where
     /// [`arithmetic weighted mean`] https://en.wikipedia.org/wiki/Weighted_arithmetic_mean
     fn weighted_mean(&self, weights: &Self) -> Result<A, MultiInputError>
     where
-        A: Copy + Mul<Output = A> + Zero;
+        A: Copy + Div<Output = A> + Mul<Output = A> + Zero;
 
-    /// Returns the [`arithmetic weighted mean`] x̅ along `axis`. Assumes that the weights are
-    /// already normalized.
+    /// Like `weighted_mean`, but assumes that the `weights` are normalized. In that case, this
```

Comment:
I would just describe what a weighted sum is and then point to the equivalence with weighted_mean if the weights are normalised.